app/javascript/src/components/Notes/CreateForm.jsx

```diff
@@ -0,0 +1,97 @@
+import React from "react";
+
+import { Form, Formik } from "formik";
+import { Check } from "neetoicons";
+import { Toastr, Pane } from "neetoui";
+import { Input, Select, Button } from "neetoui/formik";
+import * as Yup from "yup";
+
+import { TAG_DATA, CONTACT_DATA } from "./constants";
+
+const validationSchema = Yup.object({
```

Comment:
The validation schema should be moved to a `constats.js` file. This component should deal with its core business aspect. Rest of the things, which are not needed in the presentation layer should be moved out.